torchgeo/datamodules/nasa_marine_debris.py

```diff
@@ -138,3 +140,11 @@ def test_dataloader(self) -> DataLoader[Any]:
             shuffle=False,
             collate_fn=collate_fn,
         )
+
+    def plot(self, *args: Any, **kwargs: Any) -> plt.Figure:
+        """Run :meth:`torchgeo.datasets.NASAMarineDebris.plot`."""
+        try:
```

Comment:
Any chance we can simplify this? Maybe `s/dataset/self.dataset` right before we perform the split, so we can always access `self.dataset` at any time?